README.md

```diff
@@ -31,18 +39,21 @@
 
 1. execute `tests_selector_eval` in target project directory
 
-# Development
+## <a name="dev"></a> Development
+
+See [DEVELOPER.md](developer) for more info
+
+## <a name="contrib"></a> Contributing
+
+### Contributing Guidelines
+
+Read through our [contributing guidelines][contributing] to learn about our submission process, coding rules and more.
 
-- ~~Don't forget to run `make install` before you are going to try the latest changes~~ - not needed as all the changes in source files are available immediately, just call the needed command line tool from the package
+### Code of Conduct
 
-## Testing / linting with tox
+Help us keep the project open and inclusive. Please read and follow our [Code of Conduct][codeofconduct].
 
-Make sure you have Python versions 3.6, 3.7, 3.8, 3.9 (all of them). Install tox
-into your current Python environment (does not matter which one it is). Run
-`tox` to run linters/tests using different Python versions. To run for only one
-Python version, use, for example, `tox -e py36`. To check code formatting with
-black, run `tox -e py38-check-format` (code formatting check is done only with
-Python 3.8).
 
-**NB** If you have changed dependencies in `setup.py`, run `tox --recreate`
-once.
+[developer]: https://github.com/F-Secure/pytest-rts/tree/master/docs/DEVELOPER.md
+[contributing]: https://github.com/F-Secure/pytest-rts/tree/master/docs/CONTRIBUTING.md
+[codeofconduct]: https://github.com/F-Secure/pytest-rts/tree/master/docs/CODE_OF_CONDUCT.md
```

Comment:
Should these be relative links or is it better to keep them linked to `master`? Just noticed that some of the links don't work because they don't exist in master yet.

Reply:
good question, the idea is that one can use the links even if the document is copied somewhere